src/main/java/com/puppycrawl/tools/checkstyle/api/FileSetCheck.java

```diff
@@ -72,7 +72,7 @@
      * @return the sorted set of messages to be logged.
      * @throws CheckstyleException if error condition within Checkstyle occurs
      */
-    SortedSet<LocalizedMessage> process(File file, FileText fileText) throws CheckstyleException;
+    SortedSet<Violation> process(File file, FileText fileText) throws CheckstyleException;
```

Comment:
javadoc above says `The method should return the set of messages to be logged.` -> should be corrected

Reply:
Done